ADMINS.md

```diff
@@ -52,8 +52,8 @@ Adopt organizational practices documented in this repo, work in the open, and co
 
 There are currently two ways new repositories may appear in the opensearch-project organization: creating a new repo and adopting, or moving a repo from outside of the organization into it. The process is the same.
 
-The AWS Open Source Program Office (OSPO) currently owns and manages the opensearch-project organization, and has permissions to create a new repo. This will change shortly following the [formation of the OpenSearch Software Foundation](https://foundation.opensearch.org/). While the admins above have admin-level permissions, they do not have permissions to create new repositories or move repositories into the organization.
+[The OpenSearch Software Foundation](https://foundation.opensearch.org) currently owns and manages the opensearch-project organization, and has permissions to create a new repo. While the admins above have admin-level permissions, they do not have permissions to create new repositories or move repositories into the organization.
```

Comment:
> While the admins above have admin-level permissions, they do not have permissions to create new repositories or move repositories into the organization.

Is this true? I believe the admins actually do have permissions now to do this.

Reply:
@dblock how to confirm this?

Reply:
@getsaurabh02 should be able to